reliability/single-fit/rum/src/test/kotlin/com/datadog/android/rum/integration/ManualTrackingRumTest.kt

```diff
@@ -395,6 +400,217 @@ class ManualTrackingRumTest {
 
     // endregion
 
+    // region AddViewLoading time
+
+    @OptIn(ExperimentalRumApi::class)
+    @Test
+    fun `M attach view loading time W addViewLoadingTime { active view available }`(
+        @StringForgery key: String,
+        @StringForgery name: String,
+        @BoolForgery overwrite: Boolean
+    ) {
+        // Given
+        val rumMonitor = GlobalRumMonitor.get(stubSdkCore)
+        val startTime = System.nanoTime()
+        rumMonitor.startView(key, name, emptyMap())
+
+        // When
+        val endTime = System.nanoTime()
+        val expectedViewLoadingTime = endTime - startTime
+        rumMonitor.addViewLoadingTime(overwrite)
+
+        // Then
+        val eventsWritten = stubSdkCore.eventsWritten(Feature.RUM_FEATURE_NAME)
+        StubEventsAssert.assertThat(eventsWritten)
+            .hasSize(2)
+            .hasRumEvent(index = 0) {
+                // Initial view
+                hasService(stubSdkCore.getDatadogContext().service)
+                hasApplicationId(fakeApplicationId)
+                hasSessionType("user")
+                hasSource("android")
+                hasType("view")
+                hasViewUrl(key)
+                hasViewName(name)
+                hasActionCount(0)
```

Comment:
should we add assertion here that there is no loading time on this view?

Reply:
yup we could do that, good call